src/main/java/com/stripe/param/SubscriptionItemUpdateParams.java

```diff
@@ -72,6 +73,14 @@
   @SerializedName("plan")
   Object plan;
 
+  /** The ID of the price object. */
+  @SerializedName("price")
+  Object price;
+
+  /** Data used to generate a new price object inline. */
+  @SerializedName("price_data")
+  PriceData priceData;
```

Comment:
I think this is needed to move to a different price, but I'm curious if we should forbid this.

Also feels like if you use Checkout it's okay to do `price_data` but if you use SubscriptionItems you really should have Product and Price in your account already. Like how we never added `plan_data`